### PR TITLE
Add: post for t2i adapters on sdxl.

### PR DESCRIPTION
**Pull request by sayakpaul**
Doesn't need a review. Has been reviewed by @patil-suraj and the rest of the T2I Adapters team. 

Cc: @patil-suraj @hysts @@MC-E.

### CODE REVIEW

File: t2i-sdxl-adapters.md [hunk 1]
```diff
@@ -0,0 +1,186 @@
...
+You can easily try T2I-Adapter-SDXL in [this Space](https://huggingface.co/spaces/diffusers/T2I-Adapter-SDXL) or in the playground embedded below:
+
+```jsx
+<script
+	type="module"
+	src="https://gradio.s3-us-west-2.amazonaws.com/3.42.0/gradio.js"
+></script>
+
+<gradio-app src="https://diffusers-t2i-adapter-sdxl.hf.space"></gradio-app>
+```
```
Comment (sayakpaul) [inline t2i-sdxl-adapters.md:153]: @hysts this needs to be updated.

File: t2i-sdxl-adapters.md [hunk 2]
```diff
@@ -0,0 +1,186 @@
...
+
+T2I-Adapter is an efficient plug-and-play model that provides extra guidance to pre-trained text-to-image models while freezing the original large text-to-image models. T2I-Adapter aligns internal knowledge in T2I models with external control signals. We can train various adapters according to different conditions and achieve rich control and editing effects.
+
+As a contemporaneous work, ControlNet has a similar function and is widely used. However, it can be **computationally expensive** to run. This is because, during each denoising step of the reverse diffusion process, both the ControlNet and UNet need to be run. In addition, ControlNet emphasizes the importance of copying the UNet encoder as a control model, resulting in a larger parameter number. Thus, the generation is bottlenecked by the size of the ControlNet (the larger, the slower the process becomes). 
+
+T2I-Adapters provide a competitive advantage to ControlNets in this matter. T2I-Adapters are smaller in size, and unlike ControlNets, T2I-Adapters are run just for one for the entire course of the denoising process.
```
Comment (patil-suraj) [inline t2i-sdxl-adapters.md:25]:
```suggestion
T2I-Adapters provide a competitive advantage to ControlNets in this matter. T2I-Adapters are smaller in size, and unlike ControlNets, T2I-Adapters are run just ones for the entire course of the denoising process. 
```

File: t2i-sdxl-adapters.md [hunk 3]
```diff
@@ -0,0 +1,186 @@
...
+
+T2I-Adapters provide a competitive advantage to ControlNets in this matter. T2I-Adapters are smaller in size, and unlike ControlNets, T2I-Adapters are run just for one for the entire course of the denoising process. 
+
+| **Model Type** | **Model Parameters** | **Storage (fp16)** |
+| --- | --- | --- |
+| [ControlNet-SDXL](https://huggingface.co/diffusers/controlnet-canny-sdxl-1.0) | 1251 M | 2.5 GB |
+| [ControlLoRA](https://huggingface.co/stabilityai/control-lora) (with rank 128) | 197.78 M (84.19% reduction)  | 396 MB (84.53% reduction) |
+| [T2I-Adapter-SDXL](https://huggingface.co/TencentARC/t2i-adapter-canny-sdxl-1.0) | 79 M (**_93.69% reduction_**) | 158 MB (**_94% reduction_**) |
+
+Over the past few weeks, the Diffusers team and the T2I-Adapter authors have been collaborating to bring the support of T2I-Adapters for Stable Diffusion XL (SDXL) in `diffusers`. In this blog post, we share our findings from training T2I-Adapters on SDXL from scratch, some appealing results, and, of course, the T2I-Adapter checkpoints on various conditionings (sketch, canny, lineart, depth, and openpose)!
```
Comment (patil-suraj) [inline t2i-sdxl-adapters.md:33]: maybe link to diffusers github here ?

File: t2i-sdxl-adapters.md [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+- Mixed precision: fp16
+
+We encourage the community to use our scripts to train custom and powerful T2I-Adapters, striking a competitive trade-off between speed, memory, and quality. 
+
+## Using T2I-Adapter-SDXL in `diffusers`
+
+Here, we take the lineart condition as an example to demonstrate the usage of [T2I-Adapter-SDXL](https://github.com/TencentARC/T2I-Adapter/tree/XL). To get started, first install the required dependencies:
+
+```bash
+pip install git+https://github.com/huggingface/diffusers.git # for now
```
Comment (patil-suraj) [inline t2i-sdxl-adapters.md:57]:
```suggestion
pip install -U git+https://github.com/huggingface/diffusers.git
```

**Review by patil-suraj — COMMENTED** [review]
tiny nits

File: t2i-sdxl-adapters.md (hunk 2 above)
Comment (sayakpaul) [reply t2i-sdxl-adapters.md:27]:
```suggestion
T2I-Adapters provide a competitive advantage to ControlNets in this matter. T2I-Adapters are smaller in size, and unlike ControlNets, T2I-Adapters are run just once for the entire course of the denoising process. 
```

**Comment by sayakpaul** [comment]
Demo is the only thing pending before we can merge the PR. Will be done in some time.

File: t2i-sdxl-adapters.md [hunk 5]
```diff
@@ -145,14 +145,8 @@ For more details, we welcome you to check the [official documentation](https://h
 
 You can easily try T2I-Adapter-SDXL in [this Space](https://huggingface.co/spaces/diffusers/T2I-Adapter-SDXL) or in the playground embedded below:
```
Comment (sayakpaul) [inline t2i-sdxl-adapters.md:146]: @hysts this also needs an update no?

Comment (hysts) [reply t2i-sdxl-adapters.md]: Ah, you are right. Thanks for catching this!

**Pull request merged**